app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -51,7 +54,8 @@ const STEPS = [
 ]
 
 const MAIN_STEPS = STEPS.slice(0, 4)
-const SUPPLEMENTAL_STEPS = STEPS.slice(4)
+const SUPPLEMENTAL_STEPS = STEPS.slice(4, 8)
```

Comment:
Non-blocking: this seems like something that could easily break if we aren't careful.  I'm wondering if explicitly declaring them would be better for `SUPPLEMENTAL_STEPS` and `NON_VISUALIZABLE_STEPS`.

Reply:
That's a good point - I've made a follow up ticket for looking into CSFV for non-viz files and I can add this idea into that to take a look at